src/test/java/com/waverley/testcases/AuthTest.java

```diff
@@ -0,0 +1,39 @@
+package com.waverley.testcases;
+
+import com.waverley.model.User;
+import com.waverley.utils.CustomAssertions;
+import io.github.sskorol.core.DataSupplier;
+import lombok.extern.slf4j.Slf4j;
+import org.testng.ITestContext;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.lang.reflect.Method;
+
+import static com.waverley.utils.JsonUtils.jsonToEntity;
+import static java.util.Optional.ofNullable;
+
+@Slf4j
+public class AuthTest {
+
+    @DataSupplier(name = "User provider", flatMap = true)
+    public User streamOfData(Method method) {
```

Comment:
seems like you need to run `checkQualityTest` task to resolve all violations including missing `final`